Neos.Flow/Tests/Unit/ResourceManagement/ResourceTypeConverterTest.php

```diff
@@ -92,7 +92,22 @@ public function convertFromReturnsNullIfSourceArrayIsEmpty()
     public function convertFromReturnsNullIfNoFileWasUploaded()
     {
         $source = ['error' => \UPLOAD_ERR_NO_FILE];
-        $this->assertNull($this->resourceTypeConverter->convertFrom($source, PersistentResource::class));
+        $this->assertTrue($this->resourceTypeConverter->convertFrom($source, PersistentResource::class));
```

Comment:
Why was this test result changed? Did your change break it?

Reply:
Sorry, that should not be committed with, is changed now.